modules/tinymce/src/core/main/ts/api/AddOnManager.ts

```diff
@@ -267,6 +267,14 @@ const AddOnManager = <T>(): AddOnManager<T> => {
      */
     get,
 
+    /**
+     * <em>Deprecated in TinyMCE 5.7 and has been marked for removal in TinyMCE 6.0.</em>
```

Comment:
This one had been missed in previous iterations and I had noticed it was in the list, so have added this as we did for `$` in `DOMUtils`.